docs/upgrading.rst

```diff
@@ -118,15 +120,14 @@ the schema. In order to achieve this, webargs now passes the full data from
 the specified location to the schema.
 
 However, many types of request data are so-called "multidicts" -- dictionary-like
-types which can return one or multiple values as you prefer. To handle
-`marshmallow.fields.List` and `webargs.fields.DelimitedList` fields correctly,
-passing list data, webargs must combine schema information with the raw request
-data. This is done in the
+types which can return one or multiple values. To handle `marshmallow.fields.List`
+and `webargs.fields.DelimitedList` fields correctly, passing list data, webargs
+must combine schema information with the raw request data. This is done in the
 :class:`MultiDictProxy <webargs.multidictproxy.MultiDictProxy>` type, which
-will often be passed to your schemas.
+will often be passed to schemas.
```

Comment:
Should the whole paragraph be moved to line 180, with "Finally, this change passes a proxy object where schemas once saw a dict."?

Looks like this chapter interleaves unknown and multidict issues. Or maybe I read too fast.